python/samples/kernel-syntax-examples/action_planner.py

```diff
@@ -13,7 +13,7 @@ async def main():
     api_key, org_id = sk.openai_settings_from_dot_env()
 
     kernel.add_chat_service(
-        "gpt-3.5", OpenAIChatCompletion("gpt-3.5-turbo", api_key, org_id)
+        "chat-gpt", OpenAIChatCompletion("gpt-3.5-turbo", api_key, org_id)
```

Comment:
Did this need to change for this PR?

Reply:
I'm guessing it's to help test the behavior of the template.

Reply:
Yes, it's to help test the template behavior. It's harmless - just the lookup name for the service. The service itself is not changing and it's a syntax example, so no tests are affected.